### ISSUE TITLE
How do you update the global class representation?

### ISSUE THREAD

**Issue by DaiBeimin** [tiangeluo/fsl-global#2]
hello, the work in your paper is very good. I'm a student in the college.After reading this paper, I just wonder how the global class representations update in the training? Do they update in each training episode or in other way?I can not understand this point, can you tell me something about this?
Looking for your reply, thank you.

**Comment by tiangeluo** [comment]
Hi,

You can refer to Section 3.1 for details. We would update the global class representations in the training via the registration module. This is very similar to the way used in the adversarial example for updating input images. You can view the global class representations as extra inputs to our model and the gradients would backpropagate to them.
